app/src/main/res/values/strings.xml

```diff
@@ -3,4 +3,8 @@
 
     <string name="app_name">VoucherifySdk</string>
 
+    <string name="discountTitle">"Discount: %1$s"</string>
+    <string name="priceAfterDiscountTitle">"Price after discount: %1$s"</string>
+    <string name="errorVoucherMessage">"Invalid - %1$s. Try to use 'test'"</string>
```

Comment:
@ktalanda Are we sure that voucher code 'test' exists? As far as I remember we had this 'Testing7fjWdr' as it was always valid for test account.